internal/weapons/catalyst/hakushin/hakushin.go

```diff
@@ -23,21 +23,20 @@ func weapon(char core.Character, c *core.Core, r int, param map[string]int) stri
 				return false
 			}
 			m[ele] = e
-			expiry = c.F + 6*60
+
+			for _, char := range c.Chars {
+				char.AddMod(core.CharStatMod{
+					Key: "hakushin-passive-" + key,
+					Amount: func() ([]float64, bool) {
+						return m, true
```

Comment:
not sure if this is intentional or not (haven't looked into how this thing works) but because m is declared way at the top, you can get a situation where you're adding elemental dmg for multiple elements at the same time because you're not resetting the values